src/core/mount.c

```diff
@@ -1865,6 +1865,10 @@ static void mount_enumerate(Manager *m) {
                 }
 
                 (void) sd_event_source_set_description(m->mount_event_source, "mount-monitor-dispatch");
+
+                r = sd_event_source_set_ratelimit(m->mount_event_source, 1000000, 5);
```

Comment:
instead of 1000000 write USEC_PER_SEC to make this easier to read. 

also maybe write a comment next to it: `/* put a ratelimit on this event source trigger only 5 times per 1s */`